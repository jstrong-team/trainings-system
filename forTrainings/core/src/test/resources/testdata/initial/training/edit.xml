<dataset>
    <training_version/>
</dataset>