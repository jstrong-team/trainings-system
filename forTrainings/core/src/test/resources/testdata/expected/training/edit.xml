<dataset>
    <training_version old_version="1"/>
</dataset>